core/arch/arm/plat-rcar/conf.mk

```diff
@@ -0,0 +1,28 @@
+PLATFORM_FLAVOR ?= h3
+PLATFORM_FLAVOR_$(PLATFORM_FLAVOR) := y
```

Comment:
Remove this line (line 2). The variable is now set automatically in the main Makefile (I noticed some platforms still have such a line, I will create a PR to clean them up).